packs/coder-2.21.3/values.yaml

```diff
@@ -0,0 +1,357 @@
+# coder -- Primary configuration for `coder server`.
```

Comment:
```yaml
pack:
  content:
    images:
      - image: ghcr.io/coder/coder:v2.21.3
    charts:
      - repo: helm.coder.com/v2
        name: coder
        version: 2.21.3
```

You might want to put this at the top, one of the benefits of a _pack_ is the benefit to bring it into the Airgapped ready ecosystem.